webview/src/shared/components/dragDrop/DragDropContainer.tsx

```diff
@@ -189,6 +201,7 @@ export const DragDropContainer: React.FC<DragDropContainerProps> = ({
     dispatch(changeRef(undefined))
 
     onDrop?.(oldDraggedId, draggedRef?.group || '', group, droppedIndex)
+    cleanup()
```

Comment:
I don't think this should be there. I need to look into why I added this and if it was intentional.